input/about/bylaws.md

```diff
@@ -67,7 +67,17 @@ Amended and Restated Effective June 4, 2020
 
 **(e)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Limited Special Right for Director Appointed by Founding Member.** In connection with any vote to materially change the Foundation’s Membership Policy, Director Election Policy, Project Governance Policy, or any Intellectual Property-related agreements or policies, (a) a “no” vote by the director appointed by the Founding Member will result in the disapproval of a proposed action, regardless of the number of votes for approval), and (b) such director must be present as part of any quorum (i.e. if that director is not present, the Board will not have a quorum for the matter, regardless of the number of other directors present).
 
-**(f)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Notices.** All notices required under this Article III will be given to all directors in office at the time of such notice and may be given by email at least 48 hours in advance of the meeting or by first class mail to such director’s last known business address at least three business days in advance.
+**(f)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Member Review.** Prior to voting on any amendments to the Foundation’s Bylaws, Articles of Incorporation or the Foundation's Membership Policy, the Director Election Policy, the Project Governance Policy, or any of the Foundation’s Intellectual Property-related agreements or policies, the Board shall use reasonable efforts to preview the proposed amendments with the Foundation’s Members, using the following process:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(i)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;post the proposed amendments as “pull requests” on the Foundation’s Github, or through such other methodology determined by the Board that is accessible to Members for review and feedback;
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(ii)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;notify all Members via email and solicit Member comments to the proposed changes over a designated Member review period (no less than a week from the date of notice), to be submitted by Members via Github or such other forum or methodology adopted by the Board;
```

Comment:
I'm not sure we want to specify email here. Can we say something to the extent of "in a public forum" to encompass social media? A tweet calling attention to the proposed changes should be enough?

Reply:
Yes, good idea. I'm thinking "notify Members via a public announcement" instead of "notify all Members via email." Apologies for my lack of GitHub skills, but do I delete my PR, make the change in my local repository, and submit a new PR? Or can I edit this PR somehow?

Reply:
If you add more commits to the branch from the PR (master in this case), it'll show up here. So just edit and push again.

Next time (and I can show you how), you'd typically push to a branch and then create the PR from there. GitHub Desktop can help make this easier: https://desktop.github.com/

Reply:
> A tweet calling attention to the proposed changes should be enough?
While I am here via Twitter, I think if we have member emails sending an email about proposed bylaw changes is preferable. Not everyone is as active on social media and bylaw changes could fundamentally change the organization, so I think an email would be appropriate if possible.

Reply:
I edited to "public announcement," but that certainly wouldn't preclude sending email to the members, as desired.

Reply:
I think leaving it as "public announcement" is good as it shows intent. We can determine what the best mechanisms are as needed. We may need to be careful with email as we don't want to spam folks either.